sql_generators/glean_usage/templates/baseline_clients_daily_v1.metadata.yaml

```diff
@@ -19,6 +19,7 @@ bigquery:
     type: day
     field: submission_date
     require_partition_filter: true
+    expiration_days: 775
```

Comment:
Just out of curiosity, what number should be used for expiration days? I have seen 780 as well: https://github.com/mozilla/bigquery-etl/blob/b81a7a09bb8a9c1c41f3961640622190c7041da1/sql/moz-fx-data-shared-prod/firefox_ios_derived/active_users_aggregates_v1/metadata.yaml#L23

Reply:
In the recent data retention sync meeting, we discussed and decided to use 775. The above PR was before a number was finalized. Open to suggestions or discussions if needed.

Reply:
Should we change the 780s to 775s in a separate PR? To make it consistent?

Reply:
I have changed 780 to 775 for a couple of tables. This file is deprecated and I will add it to my to-be-deleted list
ql/moz-fx-data-shared-prod/firefox_ios_derived/active_users_aggregates_v1/metadata.yaml#L23